Wikipedia/assets/codemirror/resources/addon/search/search.js

```diff
@@ -129,7 +129,23 @@
         element.classList.remove(focusClassName);
         if (element.id === focusedMatchID) element.id = "";
       }
-    } 
+    }
+
+    function markReplacedText(cm, cursor) {
+      const state = getSearchState(cm);
+      const marker = cm.markText(cursor.from(), cursor.to(), { className: 'cm-searching-replaced' })
+      if (state.replacedMarkers) {
+        state.replacedMarkers.push(marker);
+      } else {
+        state.replacedMarkers = [marker];
+      }
+    }
+
+    function clearReplaced(state) {
+      if (state.replacedMarkers) {
+        state.replacedMarkers.forEach((marker) => { marker.clear() });
```

Comment:
Might want to set replacedMarkers to [] here too, otherwise it'll build up.

Reply:
fixed